README.rst

```diff
@@ -124,18 +124,33 @@ sites)::
   ./manage.py lms collectstatic
   ./manage.py cms collectstatic
 
+Setup Studio SSO for Development::
+
+  ./manage.py lms manage_user studio_worker example@example.com --unusable-password
+  # DO NOT DO THIS IN PRODUCTION it will make your auth insecure
+  ./manage.py lms create_dot_application --grant-type authorization-code --skip-authorization --redirect-uris "http://localhost:18010/complete/edx-oauth2/" --scopes "user_id" studio-sso studio_worker --client-id studio-sso-key --client-secret studio-sso-secret
+
+Setup Studio SSO for Production::
+
+  ./manage.py lms manage_user studio_worker <email@yourcompany.com> --unusable-password
+  ./manage.py lms create_dot_application --grant-type authorization-code --skip-authorization --redirect-uris "http://localhost:18010/complete/edx-oauth2/" --scopes "user_id" studio-sso studio_worker
+  # Login to the django admin site (eg. http://localhost:18000/admin/oauth2_provider/application/)
+  # to view the credentials for the 'studio-sso' app and set them in the config
+  # for your production studio deployment using the `SOCIAL_AUTH_EDX_OAUTH2_KEY`
+  # and `SOCIAL_AUTH_EDX_OAUTH2_SECRET` values in your studio settings.
+
```

Comment:
```suggestion
Set up CMS SSO (for Production):

* Create the CMS user and the OAuth application::

    ./manage.py lms manage_user cms <email@yourcompany.com> --unusable-password
    ./manage.py lms create_dot_application \
        --grant-type authorization-code \
        --skip-authorization \
        --redirect-uris "http://localhost:18010/complete/edx-oauth2/" \
        --scopes "user_id" cms-sso cms

* Log into Django admin (eg. http://localhost:18000/admin/oauth2_provider/application/),
  click into the application you created above (``cms-sso``), and copy its "Client secret".
* In your private Django settings module, set ``SOCIAL_AUTH_EDX_OAUTH2_SECRET`` to the 
  secret that you copied.
* If you used a Client ID other than ``cms-sso``, then set ``SOCIAL_AUTH_EDX_OAUTH2_KEY``
  to your Client ID in your private Django settings module as well.
```

* `studio_worker` -> `cms`
* `studio-sso` -> `cms-sso`
* break up the long shell lines for readability
* I think your code comment would be easier to follow if spelled out as bullet points outside the code block.
* Rather than telling them to set ``SOCIAL_AUTH_EDX_OAUTH2_KEY``, why not set it for them in minimal.yml?